Doc/library/pathlib.rst

```diff
@@ -539,6 +539,18 @@ Pure paths provide the following methods and properties:
       ValueError: '/etc/passwd' does not start with '/usr'
 
 
+.. method:: PurePath.is_relative_to(*other)
+
+    Compute the boolean result that this path is relative to the path
+    represented by *other* or not.
+
+      >>> p = PurePosixPath('/etc/passwd')
```

Comment:
I would suggest changing `PurePosixPath` to `PurePath`, `PurePath` will use `PurePosixPath` or `PureWindowsPath` based on the environment (for windows compatibility), and adding the import statement so that users can directly run the interpreter example.
```suggestion
      >>> from pathlib import PurePath
      >>> p = PurePath('/etc/passwd')
```

Reply:
you are right, show the PurePath demo would be better to user.